server/src/models/Match.ts

```diff
@@ -0,0 +1,76 @@
+import { getSettings } from '@src/services/SettingsService'
+
+export class Match {
+  constructor (
+    public matchKey: string,
+    public eventKey: string,
+    public compLevel: 'qm' | 'sf' | 'f',
+    public setNumber: number,
+    public matchNumber: number,
+
+    public startTime: EpochTimeStamp,
+    public endTime: EpochTimeStamp,
+
+    public blue_keys: string[],
+    public red_keys: string[],
+
+    public blueScore: number,
+    public redScore: number,
+
+    public winner: 'blue' | 'red' | ''
+  ) { }
+
+  private static readonly compLevelString = {
+    qm: 'Qualification Match',
+    // qf: 'Quarterfinals Match',
+    // sf: 'Playoffs Match',
+    f: 'Finals Match'
+  }
+
+  public getMatchName (): string {
+    if (this.compLevel === 'sf') return `Playoffs Match ${this.setNumber}`
+
+    return `${Match.compLevelString[this.compLevel]} ${this.matchNumber}`
+  }
+
+  private getAllianceString (keys: string[]): string {
+    return keys.reduce((a, b) => a + ', ' + b.substring(3)).substring(3)
+  }
+
+  // hardcode as a test for now...
+  public async genVideoDescription (): Promise<string> {
```

Comment:
Kind of an overall sentiment for this file: I would prefer that we don't necessarily implement all these stub methods yet so we aren't digging ourselves into an architectural hole. This is a lot of business logic to be including in a model class.

For this PR, I'd prefer we comment/remove the code and implement it more robustly as part of upcoming work.